doc/app-flows.md

```diff
@@ -0,0 +1,33 @@
+# App Flows
+
+## Main App Flow
+
+1. User receives a text with a redemption link.
+
+2. Check if the customer is logged in with `Get Current Customer`.
```

Comment:
It seems like 2 & 4 could be in it's open section on `Logging In and Creating Accounts`.